srcpkgs/modrana/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'modrana'
+pkgname=modrana
+version=0.57.3
+revision=1
+# releases aren't tagged in git
+_version_commit=dbd558d407abe7a3a4a352c96b618147cade4bd9
+wrksrc="modrana-${_version_commit}"
+hostmakedepends="rsync python3 qt5-qmake"
+makedepends="qt5-declarative-devel"
+depends="pyotherside"
+short_desc="Flexible GPS navigation system for mobile devices"
+maintainer="Joel Beckmeyer <joel@beckmeyer.us>"
+license="GPL-3.0-or-later"
+homepage="https://modrana.org"
+distfiles="https://github.com/M4rtinK/modrana/archive/${_version_commit}.tar.gz"
+checksum=7f93e4660997a8cba4194709c4566dec667da1c99ae45dd66ea6df71b8c637a6
+python_version=3
+
+do_build() {
+	make VersionFile
+	make
+	make rsync
+	make bytecode-python3
+	make launcher
```

Comment:
this could maybe be `make VersionFile all rsync bytecode-python3 launcher`? (not sure if this will work due to parallelism)

also this might be able to be done with one of the make buildstyles?

Reply:
Thanks, took a little bit of fiddling, but I got it to use `build_style=gnu-makefile`